test/extended/authorization/scc.go

```diff
@@ -164,3 +164,56 @@ var _ = g.Describe("[sig-auth][Feature:PodSecurity][Feature:SCC]", func() {
 		o.Expect(pod.Annotations[securityv1.ValidatedSCCAnnotation]).To(o.Equal("restricted-v2"))
 	})
 })
+
+var _ = g.Describe("[sig-auth][Feature:PodSecurity][Feature:SCC]", func() {
+	defer g.GinkgoRecover()
+
+	oc := exutil.NewCLIWithPodSecurityLevel("required-scc", psapi.LevelPrivileged)
```

Comment:
How about just putting this g.It within previously existing `g.Describe("[sig-auth][Feature:PodSecurity][Feature:SCC]", func() {` ?

Reply:
Thanks, Fixed!